packages/tatum-core/src/model/request/ActiveBuyTrades.ts

```diff
@@ -0,0 +1,79 @@
+import { Type } from 'class-transformer'
```

Comment:
Just wondering if it wouldnt be better to create one base Trade class where the most properties would be defined. Than ActiveBuyTrades, ActiveSellTrades and HistoricalTrades will inherite these properties.

Reply:
yeah it makes sense, the only difference is in one property. I don't know why I didn't do it already, thanks!